docs/pages/includes/kubernetes-access/rbac.mdx

```diff
@@ -22,7 +22,7 @@ spec:
     kubernetes_labels:
       '*': '*'
     kubernetes_groups:
-    - view
+    - system:masters
```

Comment:
Is there an alternative we can use to `system:masters`? The original partial used `view` to follow the principle of least privilege.

Reply:
That's the only thing I didn't like about using `system:masters`. I'll look into other default group/user binds. If there isn't a suitable default then we'll have to include instructions for creating a bind

Reply:
Depending on how the partial looks within the docs pages that include it, I think we could either:
- Add instructions for creating a bind—I'm guessing there are a lot of readers who are pretty familiar with Kubernetes fundamentals but could use a briefing on RBAC best practices.
- Use an obvious placeholder value for the group, e.g., `my-group`, with a note that the user will need to have defined a `Role` and `[Cluster]RoleBinding` already for that group.

Reply:
We could leave the teleport role snippet using the "view" kuberenetes_group and provide this snippet for them to use:

```
$ echo "                                                                                                                                                                                                         (base) 
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRoleBinding
  metadata:
    name: my-bind
  subjects:
  - kind: Group
    name: system:authenticated
    apiGroup: rbac.authorization.k8s.io
  roleRef:
    kind: ClusterRole
    name: view
    apiGroup: rbac.authorization.k8s.io
  " | kc apply -f -
```

The default clusterrole view that this binding uses has these permissions:

> Allows read-only access to see most objects in a namespace. It does not allow viewing roles or role bindings.
>
> This role does not allow viewing Secrets, since reading the contents of Secrets enables access to ServiceAccount credentials in the namespace, which would allow API access as any ServiceAccount in the namespace (a form of privilege escalation).

Reply:
I tested that it works for doing `kubectl get pods` as suggested in our getting-started guide.

Reply:
That sounds good to me!

Reply:
Changed, if you could take a look again